spec/lib/efile/relationship_spec.rb

```diff
@@ -45,4 +45,8 @@
       end
     end
   end
+
+  describe "qualifying_relative_requires_member_of_household_test?" do
```

Comment:
This branch still needs some testing. If you think its helpful to have this stuff, though, in order to get going on confirm_dependents, we can chunk it.